### ISSUE TITLE
Automated Web Accessibility Testing Tools Are Not Judges

### ISSUE THREAD

**Issue by studiomohawk** [frontendweekly/feeds#1698]
[Automated Web Accessibility Testing Tools Are Not Judges](http://ift.tt/2nvouom) / Recently social media has been abuzz regarding an article titled "ITIF: 92% of Top Federal Websites Fail to Meet Security, Speed, Accessibility Standards" &ndash; and&hellip;
